pybit/_websocket_stream.py

```diff
@@ -176,11 +225,12 @@ def resubscribe_to_topics():
                 self.exit()
                 raise websocket.WebSocketTimeoutException(
                     f"WebSocket {self.ws_name} ({self.endpoint}) connection "
-                    f"failed. Too many connection attempts. pybit will no "
-                    f"longer try to reconnect."
+                    "failed. Too many connection attempts. pybit will no "
+                    "longer try to reconnect."
+
                 )
 
-        logger.info(f"WebSocket {self.ws_name} connected")
+        logger.info("WebSocket %s connected", self.ws_name)
```

Comment:
Is this just a style change or is this a better way? Personally prefer `{}`...

Reply:
Forgot to comment this change. Yes, it is not about the style, it's about performance. f-strings will format the message even if it's not logged. In case of %s strings and arguments passed through `*args`, the string will be formatted when the message is logged(and, therefore, will not be formatted if the message isn't logged). So we save the resources.